cli/sawtooth_cli/identity.py

```diff
@@ -24,6 +24,11 @@
 import random
 import yaml
 
+from sawtooth_signing import create_context
```

Comment:
Is this file change a lint correction? Similarly, next two files. I see change in the order import statements are written.

Reply:
Yep, since we're installing these modules from packages, lint prefers them to be imported in a different order.